docs/admin/federation-apiserver.md

```diff
@@ -1,8 +1,3 @@
 This file is autogenerated, but we've stopped checking such files into the
 repository to reduce the need for rebases. Please run hack/generate-docs.sh to
 populate this file.
-
```

Comment:
Yes, two commits are the way we do it. I was also investigating those documentation file changes and I confirm it happens for me too. Will investigate more to find the root cause.